adhocracy4/polls/models.py

```diff
@@ -70,17 +70,6 @@ def clean(self, *args, **kwargs):
                         )
                     }
                 )
-            elif Choice.objects.filter(question=self.pk).count() > 0:
-                raise ValidationError(
-                    {
-                        "is_open": _(
-                            "Question with choices cannot become "
-                            "open question. Delete choices or add new "
-                            "open question."
-                        )
-                    }
-                )
-
         super().clean(*args, **kwargs)
```

Comment:
@goapunk this check here is causing errors to save a new open questions instances, as the choices can only be added after a question is created. For existing questions, a check if a question is open happens when creating the choices below in the Choice model

Reply:
hmm, actually in my installed version (earlier version for 4.2 upgrade) was referring to choice as self.choices. But the code in main, looks fine.